portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1092,6 +1092,55 @@ definition {
 		PortletEntry.publish();
 	}
 
+	@description = "This is a use case for LPS-110450."
+	@priority = "5"
+	test VerifyErrorMessageInSaas {
+		property osgi.app.includes = "saml-saas";
```

Comment:
I thought `saml-saas` would be distributed somehow so I suggested building it as an LPKG (related: https://liferay.slack.com/archives/GLBT9FL23/p1596220342094500).`osgi.modules.includes` is the way to go in this case. I just tested it and it deploys the module successfully.

Reply:
Thanks, I will modify this part.